thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -539,11 +539,11 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
                     }
                 }
                 // places logical filename directory in 'dir'
-                makePhysicalPartName(logicalName, 0, 0, dir, false, os, planeDir.str());
+                getLFNDirectoryUsingBaseDir(dir, logicalName, planeDir.str());
```

Comment:
Containerized builds now fail with:

#16 139.0 /hpcc-dev/HPCC-Platform/thorlcr/mfilemanager/thmfilemanager.cpp: In member function ‘virtual IFileDescriptor* CFileManager::create(CJobBase&, const char*, StringArray&, IArrayOf<IGroup>&, bool, unsigned int, bool, unsigned int)’:
#16 139.0 /hpcc-dev/HPCC-Platform/thorlcr/mfilemanager/thmfilemanager.cpp:481:20: error: variable ‘os’ set but not used [-Werror=unused-but-set-variable]
#16 139.0   481 |             DFD_OS os = DFD_OSdefault;
#16 139.0       |                    ^~